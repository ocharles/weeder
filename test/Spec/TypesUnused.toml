roots = []

type-class-roots = false

unused-types = true
