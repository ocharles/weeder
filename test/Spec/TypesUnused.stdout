main: test/Spec/TypesUnused/TypesUnused.hs:6:1: Modulo1
main: test/Spec/TypesUnused/TypesUnused.hs:8:1: Number
main: test/Spec/TypesUnused/TypesUnused.hs:10:1: Vector
main: test/Spec/TypesUnused/TypesUnused.hs:12:1: Family
